twiggy/tests/all/expectations/elf_top_25_hello_world_rs

```diff
@@ -1,25 +1,25 @@
  Shallow Bytes │ Shallow % │ Item
-───────────────┼───────────┼────────────────────────────
+───────────────┼───────────┼───────────────────────────────────────────────────────────────
           9751 ┊     0.46% ┊ stats_arena_print
-          9503 ┊     0.45% ┊ output
+          9503 ┊     0.45% ┊ std::sys_common::backtrace::output::hf6421f76165dc3d9
```

Comment:
and here's a nice example of what we gain. as the kids say, _"you love to see it."_